python/src/scipp/_math.py

```diff
@@ -124,3 +124,60 @@ def log10(x: VariableLike, *, out: Optional[VariableLike] = None) -> VariableLik
     :returns: Base 10 logarithm of the input.
     """
     return _call_cpp_func(_cpp.log10, x, out=out)
+
+
+def around(x: VariableLike,
+           decimals: int = 0,
+           *,
+           out: Optional[VariableLike] = None) -> VariableLike:
+    """
+    Round to the nearest integer if no decimals is provided, otherwise round to the
+    nearest decimal given.
+
+    Note: if the number being rounded is halfway between two integers/decimals it
+    will round to the nearest even number. For example 1.5 and 2.5 will both round
+    to 2.0, -0.5 and 0.5 will both round to 0.0.
+
+    :param x: Input data.
+    :param decimals: Number of decimal places to round to.
+    :param out: Optional output buffer.
+    :returns: Rounded version of the data passed to the decimals given, if given,
+    else to the nearest integer.
+    """
+    if out is not None:
+        # Copy x to out, if out != x.
+        out[...] = x
+    else:
+        out = x
+    multiplier = 10.0**decimals
+    out = out * multiplier
+    _cpp.round(out, out=out)
+    out /= multiplier
+    return out
```

Comment:
This bit in the wrapper appears to be untested?

Reply:
Yeah I didn't get round to it yesterday before end of work, will do it today :+1:

Reply:
Done :+1: